examples/example-server-lib/sw_splash.cpp

```diff
@@ -143,6 +142,8 @@ struct SwSplash::Self : SplashSession
 
     Globals globals;
 
+    bool splash;
+
```

Comment:
Probably better to avoid default initialize for a bool. Vis:
```C++
    bool splash = true;
```
And maybe better to call it `show_splash`.

Reply:
fixed in pr update